src/app/components/types/tree/tree.component.html

```diff
@@ -0,0 +1,109 @@
+    <mat-divider></mat-divider>
+    <mat-card-content>
+      <mat-tab-group mat-stretch-tabs dynamicHeight>
+        <mat-tab>
+          <ng-template matTabLabel>Demo</ng-template>
+          <td-chart [style.height.px]="800"
+          [config]="{ xAxis: { show: false }, yAxis: { show: false }, grid: { borderColor: 'transparent' } }">
+            <td-chart-tooltip [trigger]="'item'">
+              <ng-template let-params let-ticket="ticket" tdTooltipFormatter>
+                <ng-container *ngIf="params">
+                  <div layout="row" layout-align="start center">
+                    <mat-icon class="push-right-sm">
+                      <span class="tc-blue-300"> info</span>
+                    </mat-icon>
+                    <span> {{params.name}} </span>
+                  </div>
+                </ng-container>
+              </ng-template>
+            </td-chart-tooltip>
+            <td-chart-series td-tree
+                            [top]="'10%'"
+                            [left]="'10%'"
+                            [bottom]="'10%'"
+                            [right]="'10%'"
+                            [data]="data"
+                            [initialTreeDepth]="2"
+                            [symbolSize]="10"
+                            [itemStyle]="{ borderColor: '#3A4950', color: '#3A4950' }"
+                            [leaves]="{ label: { padding: [2.5, 5], fontSize: 9, distance: 5, position: 'right', backgroundColor: 'transparent', color: '#616d73' } }"
+                            [label]="{ backgroundColor: '#3A4950', color: 'white', padding: [2.5, 5], borderRadius: 10, fontSize: 9, distance: 5, position: 'left' }">
+            </td-chart-series>
+          </td-chart>
+        </mat-tab>
+        <mat-tab>
+          <ng-template matTabLabel>Code</ng-template>
+          <p>HTML: </p>
+          <td-highlight lang="html">
+            <![CDATA[
+            <td-chart [style.height.px]="800"
+            [config]="{ xAxis: { show: false }, yAxis: { show: false }, grid: { borderColor: 'transparent' } }">
+              <td-chart-tooltip [trigger]="'item'">
+                <ng-template let-params let-ticket="ticket" tdTooltipFormatter>
+                  <ng-container *ngIf="params">
+                    <div layout="row" layout-align="start center">
+                      <mat-icon class="push-right-sm">
+                        <span class="tc-blue-300"> info</span>
+                      </mat-icon>
+                      <span> { {params.name} } </span>
+                    </div>
+                  </ng-container>
+                </ng-template>
+              </td-chart-tooltip>
+              <td-chart-series td-tree
+                              [top]="'10%'"
+                              [left]="'10%'"
+                              [bottom]="'10%'"
+                              [right]="'10%'"
+                              [data]="data"
```

Comment:
realizing it will be a lot of inline code, but can you pull the data into the pure html demo like the other charts? you can see the scatter chart demo is pretty long https://github.com/Teradata/covalent-echarts/blob/develop/src/app/components/types/scatter/scatter.component.html#L33-L52 which is ok